src/services/user/services/forcePasswordChange.js

```diff
@@ -0,0 +1,37 @@
+const { BadRequest } = require('@feathersjs/errors');
+const { passwordsMatch } = require('../../../utils/passwordHelpers');
+
+const setupNewPasswordProvidedByUser = (app) => async (data, params) => {
+	const newPassword = data['password-1'];
+	if (!passwordsMatch(newPassword, data['password-2'])) {
+		return Promise.reject(new BadRequest('Die neuen Passwörter stimmen nicht überein.'));
+	}
+
+	const { accountId } = params.authentication.payload;
```

Comment:
If params.account._id is exist at this point i will prefer the way.

Reply:
Not sure if I understood it correctly.
So should I check here something like
```
if (!accountId) {
   return Promise.reject(new BadRequest('Account Id not provided');
}
```
?

Reply:
No check.
You extract the accountId from the payload. But the save way to get it is from params.account._id.

Reply:
ok I see, that you have fixed that. Thanks